test/ArraySerializableTest.php

```diff
@@ -14,6 +14,7 @@
 use Zend\Hydrator\Exception\BadMethodCallException;
 use Zend\Hydrator\ArraySerializable;
 use ZendTest\Hydrator\TestAsset\ArraySerializable as ArraySerializableAsset;
+use ZendTest\Hydrator\TestAsset\ArraySerializableNoGetArrayCopy as ArraySerializableAssetNoGetArrayCopy;
```

Comment:
Instead of importing this, just use it as `TestAsset\ArraySerializableNoGetArrayCopy` when you need it within the test file.